games.yaml

```diff
@@ -1663,23 +1663,29 @@
       added: 2015-04-11
       url: http://spiderwebforums.ipbhost.com/index.php?/forum/12-blades-of-exile/
       repo: https://github.com/calref/cboe
-      info: active development, C++, GPL2
+      info: active development
+      lang: C++
+      license: GPL2
 
 - names:
     - [Exile 1988, Exile (1988 video game)]
   clones:
     - name: EXILE
       added: 2015-06-22
       repo: https://github.com/Geti/EXILE
-      info: halted development, (LÖVE, Lua), custom (AS-IS) licence
+      status: halted development
```

Comment:
`status: halted` instead of `status: halted development`

most of the other changes in this PR could drop the 'development' part from the `status` key.